build.gradle

```diff
@@ -1,34 +1,30 @@
 plugins {
-	id 'java-gradle-plugin'
-	id 'com.gradle.plugin-publish' version '0.11.0'
+	id 'com.gradle.plugin-publish' version '1.1.0'
 }
 
 repositories {
 	mavenCentral()
-	jcenter()
 }
 
 dependencies {
 	implementation 'org.apache.commons:commons-lang3:3.9'
-	implementation 'commons-io:commons-io:2.6'
-	implementation 'org.apache.velocity:velocity-engine-core:2.2'
+	implementation 'commons-io:commons-io:2.7'
+	implementation 'org.apache.velocity:velocity-engine-core:2.3'
 }
 
-pluginBundle {
-	website = 'https://bitbucket.org/cpointe/flyway-java-migration-checksum-gradle-plugin'
-	vcsUrl = 'https://bitbucket.org/cpointe/flyway-java-migration-checksum-gradle-plugin.git'
-	tags = ['flyway']
-}
-
-version = '1.0.0'
+group = 'org.bitbucket.cpointe'
```

Comment:
S: `org.technologybrewery`?

Reply:
Yeah been communicating with the Gradle Plugin Portal team on the way ahead here - was hoping to get these updates published to support a quick turnaround need without manual review, but it looks like it's being reviewed and will require manual approval.  I'll just go ahead and get the plugin updated with the new group ID references.